docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc

```diff
@@ -0,0 +1,171 @@
+:toc: macro
+== 32.03 Release Notes
+
+The following table outlines the release particulars:
+
+[cols="1,4"]
+|===
+|Build
+|32.02.127 TODO:
+|Code name
+|O'Neal - Update 2, 32.02 TODO:
+
+|Release date
+|January 28th, 2024 TODO:
+
+|Type
+|Maintenance Release
+
+|SHA-256 TODO:
+|a4790f3e94509fb1d80b0ff5a3567ac57295fd52b976b7f67d34ab268f9636ee
+
+|===
+
+Review the https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements] to learn about the supported operating systems, hypervisors, runtime, tools, and orchestrators.
+
+// You can download the release image from the Palo Alto Networks Customer Support Portal, or use a program or script (such as curl, wget) to download the release image directly from our CDN:
+
+// LINK
+
+toc::[]
+
+[#upgrade]
+=== Upgrade from Previous Releases
+
+[#upgrade-defender]
+==== Upgrade Defender Versions 22.06 and Earlier
+
+With the `v32.00` release, https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/welcome/support_lifecycle[Defender versions supported (n, n-1, and n-2)] are `v32.00`, `v31.00`, and `v30.00`.
```

Comment:
https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/welcome/support_lifecycle Please check. It throws 404 by me.

Reply:
@smitapaloalto 
I suspect the same issue here. Can you confirm that this link works:
https://docs.prismacloud.io/en/compute-edition/32/admin-guide/upgrade/support-lifecycle